target/linux/ramips/dts/mt7620a_zyxel_emg1702-t10a.dts

```diff
@@ -0,0 +1,58 @@
+#include "mt7620a_tbs.dtsi"
```

Comment:
Do you plan to add other similar devices?
Creating a dtsi for just one device is not really helpful ...

Reply:
Yes... TBS is used in DLink DIR-815 and DIR-819.
I plan to release both as separate commits after that... Already tested this dtsi with both....